hw/formal/tools/jaspergold/jaspergold.hjson

```diff
@@ -7,9 +7,10 @@
                "-proj jgproject",
                "-allow_unsupported_OS"]
 
-  // If this line is not seen in the log, then fail the test.
-  build_pass_patterns: ["^INFO: Proof threads stopped\\.$"]
-
+  build_fail_patterns: [// FuseSoC build error
+                        "^ERROR:.*$",
+                        // Jaspergold tool error
+                        ".*jg: command not found.*"]
```

Comment:
Well, this is not specific to `jg` - its a linux system error, i.e. if you run `foo` in your terminal, it will say `foo: command not found`. This should have been caught by `sim.mk` since it would have returned a non-zero exit code.